garrysmod/lua/vgui/dbutton.lua

```diff
@@ -40,6 +40,9 @@ function PANEL:SetImage( img )
 		return
 	end
 
+	self:SetColorIcon()
+	self:SetAvatarIcon()
+

```

Comment:
Surely we can avoid calling both of them? They both do the exact same thing - eventually call back to this exact function.